front-end/src/views/update_card/UpdateCard.jsx

```diff
@@ -28,38 +34,55 @@ function UpdateCard() {
         userId,
       })
       .then((res) => {
+        setRedirect(true);
         return res["data"];
       })
       .catch((err) => {
         console.log(err);
       });
   };
 
+  if (redirect) {
+    return <Redirect to={`/deck/${deckId}`} />;
+  }
+
+  //extract 'templateData' to pass to cardEditorProps so that deck specific fields aren't editable
+  const templateData = {
+    "sectionLabel0": form["sectionLabel0"],
+    "sectionLabel1": form["sectionLabel1"],
+    "sectionLabel2": form["sectionLabel2"],
+    "sliderLabelMin": form["sliderLabelMin"],
+    "sliderLabelMax": form["sliderLabelMax"]
+  };
+
   const prompt = (
     <p>
-      Edit your card so your {deckId} teammates can get the best information
+      Edit your card so your teammates can get the best information
       about you!
     </p>
   );
 
   const btn = (
-    <NavLink to={`/deck/${deckId}`}>
       <Button
         btnText="Save card changes to deck"
         onClick={() => {
-          saveCard();
+          saveCard();      
         }}
         icon={<Icon.ArrowRight />}
       />
-    </NavLink>
+    
   );
 
-  return (
+  const shouldDisableTour = true;
+
+  return !isCardLoaded ? (
+    <LoadingSpinner />
+  ) : (
     <CreateBody
       header="Update Your Card"
       prompt={prompt}
       btn={btn}
-      cardEditorProps={{ form, setForm }}
+      cardEditorProps={{ shouldDisableTour, templateData, form, setForm }}
```

Comment:
no need for line 76 if defined like this
```suggestion
      cardEditorProps={{ shouldDisableTour: true, templateData, form, setForm }}
```